specs/glossary.md

```diff
@@ -483,6 +484,14 @@ When submitted to Ethereum calldata, the batcher transaction's receiver must be
 transaction must also be signed by a recognized batch submitter account. The recognized batch submitter account
 is stored in the [System Configuration][system-config].
 
+## Batch submission frequency
+
+Within the [sequencing-window] constraints the batcher is free to submit L2 blocks to for data-availability at any time. The batcher may be reconfigured with a new policy at any time by its operator. The rollup enforces safety guarantees and liveness through the sequencing window, if the batcher does not submit data within this allotted time.
```

Comment:
```suggestion
Within the [sequencing-window] constraints the batcher is permitted by the protocol to submit L2 blocks to for data-availability at any time. The batcher software allows for dynamic policy configuration by its operator. The rollup enforces safety guarantees and liveness through the sequencing window, if the batcher does not submit data within this allotted time.
```

Reply:
Bit of a nit, but I think this is a little more neutral/objective way to say it. This could be restricted to only "safe" values (e.g. Slipper's suggestions in notion) in the std-config; so "free to" might be a misnomer in that context.